java-tools/OsmAndMapCreatorUtilities/src/main/java/net/osmand/wiki/wikidata/OsmWikiMap.java

```diff
@@ -0,0 +1,168 @@
+package net.osmand.wiki.wikidata;
```

Comment:
Generic request add support for multipolygons parsing - probably by 2 times pass  such as (https://www.openstreetmap.org/relation/8408#map=12/48.5884/9.0623)